src/server/handler/spHandler.ts

```diff
@@ -16,16 +16,18 @@ export default class SpHandler extends Handler {
 
     private async getAll(req: express.Request, res: express.Response): Promise<void> {
         try {
-            const apps = await App.find({userId: req.query.token});
+            // @ts-ignore
```

Comment:
why is this here? consider using https://github.com/hopperteam/hopper-server/blob/master/src/handler/authMiddleware.ts#L10-L16